test.js

```diff
@@ -1,13 +1,43 @@
+jest.mock("fs");
+jest.mock("find-parent-dir");
+
 var importer = require('./index');
+var fs = require('fs');
+var findParentDir = require('find-parent-dir');
+
+beforeEach(function() {
+  fs.existsSync.mockReturnValue(true);
+  findParentDir.sync.mockReturnValue('test');
```

Comment:
can we use a better directory name here. make it explicit, such as "MOCK_PARENT_DIR" or something